h2o/sample-script.py

```diff
@@ -3,7 +3,7 @@
 from pysparkling import *
 
 spark = SparkSession.builder.appName("SparklingWaterApp").getOrCreate()
-hc = H2OContext.getOrCreate(spark)
+hc = H2OContext.getOrCreate()
```

Comment:
Should we revert this?

Reply:
I didn't think this was necessary but I'll add it back and see if that helps.

Reply:
It still failing, maybe try to increase cluster nodes size? By default they are configured to `e2-standard-2` machine type.

Reply:
Done.

Reply:
Still does not work, maybe let's ignore all image versions except 2.0?

Reply:
Work for me, I'll make the change

Reply:
Just pushed

Reply:
Seems like it does not work even on 2.0

Reply:
Seemed like it was a "dataproc internal error", maybe a flake? This has consistently passed for me locally on 2.0.

Reply:
Seems like presubmit passed for all image version except 1.5 after https://github.com/GoogleCloudDataproc/initialization-actions/pull/811